### ISSUE TITLE
Does it work for the description of an issue?

### ISSUE THREAD

**Issue by soulfly** [arkhitech/redmine_mentions#20]
Does it work for the description of an issue?

For notes/comments it works, what about the description of an issue?

**Comment by jasonzzz** [comment]
Yes, https://github.com/arkhitech/redmine_mentions/pull/11/files
